### DISCUSSION TITLE
Where to get the visual theme editor?

### DISCUSSION THREAD

**Discussion by ghost** [primefaces/community#1384] [PrimeReact] [answered]
Hello!

We were thinking of buying the theme designer to create our own style based on the arya-orange theme.

I noticed that the theme designer is no longer usable: https://designer.primereact.org/

Apparently it became [open source](https://www.reddit.com/r/reactjs/comments/11rtnef/primereact_theme_designer_is_now_free_and_open/) last year, however the [repository](https://github.com/primefaces/primereact-sass-theme) seems to only contain the themes but not the visual editor itself.

I found that there is a [visual editor]( https://www.primefaces.org/designer-jsf/) for JSF (not React but ok...) but it seems to only accept selected themes (and not the arya-orange and others).

How can I get access to the PrimeReact theme designer today?

**Comment by yvonne910rhodes** [comment]
The PrimeReact theme designer is not directly accessible as it was before. To create your own theme, you can download the release matching your PrimeReact version and access the themes/mytheme/[MyMilestoneCard login](https://www.my-milestonecard.com/) folder to customize the SASS variables. For a dynamic approach, PrimeReact plans to utilize CSS variables instead of SCSS in the near future. For more detailed information, you may refer to the official PrimeReact theming documentation.

**Comment by sja-cslab** [comment]
If you have not noticed yet, Prime is replacing SCSS with CSS see [Roadmap](https://primereact.org/roadmap/).
Furthermore it seems to be part of PRO soon.
![image](https://github.com/primefaces/community/assets/129374760/833771fc-9202-4493-a454-75968d74d8a7)

**Comment by ghost** [comment]
@yvonne910rhodes @sja-cslab 

I understand that Prime React is making changes to the css infrastructure, in fact I think it's great.

What I don't understand is the decision to remove the current Theme Designer completely until the new one is released.

1. The new one is not finished yet.
2. If you already promoted it in a previous point, why remove it all of a sudden, to many users who still rely on it?

And, by the way, what will be part of the PRO is the "Visual Theme Builder" not the "Visual Theme Designer" as this image says, which also says that with PRO you will have an extended license.

![image](https://github.com/primefaces/community/assets/71069615/68040276-2275-45e0-96a9-ca2a801dc571)

**Reply by sja-cslab** [reply, marked as answer]
I agree with you and i dont know why its done that way. I just tried to give you all information I have.
What we do normally is, taking a default theme from prime and customize it that may work for you too.
